fixed-requirements.txt

```diff
@@ -27,7 +27,7 @@ passlib==1.7.1
 lockfile==0.12.2
 python-gnupg==0.4.2
 jsonpath-rw==1.4.0
-pyinotify==0.9.6
+git+https://github.com/blag/watchdog3.git#python2.7-support
```

Comment:
Can we please move it to `StackStorm` org?

A lot of issues when we're creating forks under own github user account, instead of org.

Reply:
As per my comment above - it looks like the upstream project is active so I would just wait for @blag's changes to me merged and then pull directly from a specific revision from the upstream repo (it will probably take a while until they publish new version to PyPi, but maybe @blag can poke them :)).

Reply:
+1 for one way or another that will remove presence of personal github accounts